lib/sprockets/commoner.rb

```diff
@@ -13,6 +13,10 @@ module Commoner
   register_bundle_metadata_reducer 'application/javascript', :commoner_required, Set.new, :+
   register_bundle_metadata_reducer 'application/javascript', :commoner_used_helpers, Set.new, :+
   register_bundle_processor 'application/javascript', ::Sprockets::Commoner::Bundle
+  register_dependency_resolver 'commoner-environment-variable' do |env, str|
+    _, variable = str.split(':', 2)
+    ENV[variable]
+  end
```

Comment:
@rafaelfranca this would actually be a useful dependency resolver to have built into sprockets itself